packages/atomic/cypress/integration/common-assertions.ts

```diff
@@ -57,19 +58,8 @@ export function assertRemovesComponent(
   });
 }
 
-export function assertAriaLiveMessage(
-  selector: () => Cypress.Chainable<JQuery<HTMLElement>>,
-  message: string
-) {
+export function assertAriaLiveMessage(message: string) {
   it(`screen readers should read out "${message}".`, () => {
-    selector().should('contain.text', message);
-  });
-}
-
-export function assertNoAriaLiveMessage(
```

Comment:
I removed the `assertNoAriaLiveMessage` tests since I'm afraid they would become flaky due to a variety of elements that can update the `aria-live` text.